src/components/CodeEditor.js

```diff
@@ -75,6 +77,74 @@ class CodeEditor extends Component {
       }
     })
 
+    // Track current mouse position in the code
+    window.addEventListener('mousemove', e => {
```

Comment:
Should we limit the event listener to the code editor dom node? Or maybe even let react handle the listening?

Reply:
yeah thats probably a good idea. I'll make the change

Reply:
Oh also we should remove it on `componentWillUnmount`.

Reply:
ah yes. Good point.